client/macro_processor.py

```diff
@@ -125,7 +128,7 @@ def postprocess_file(self, tmp_path: str, output_path: str):
         print("Postprocessing into %s" % output_path)
         with open(tmp_path) as tmp_fh:
             text = tmp_fh.read()
-        text = self.postprocess_text(text, output_path)
+        text = self.postprocess_text(text, self.__get_path_in_tmp_output(tmp_path))
```

Comment:
The path passed here was meant to be advisory, so the user knows which file is being processed in case that matters to their processing. It would seem to make more sense to pass the authoritative input/output path than the temp path. What do you think? Are we using the path in some way that makes us want to expose the temp path to the user? The user shouldn't be doing any I/O on that file. They might, however, want to read a sibling file which isn't in the temp directory.

Reply:
Makes sense.  Done. I updated this part a little to stay away from the temp path. 
The tricky thing is that we need to use the file path that is relative to the config input/output directory (not cwd) so that the wildcard matching works for sub-directories, e.g, subdir_1/subdir_2/*.sql.